### PR TITLE
Add CONTRIBUTING

### PR DESCRIPTION
**Pull request by ykubota**

### CODE REVIEW

**Comment by mikakadevo2** [comment]
Preview: http://ntt-tech-conference-01.s3-website-ap-northeast-1.amazonaws.com/ad817e61970086f4342c3f29dcf5d9bce271f0b5/

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How to contribute this repository?
+
+Contributions are always welcome! If you have good ideas for improvements,
+please follow the guidelines to pull requests smoothly.
+
+## Pull requests
+* Perform all changes on a topic branch.
+* Modify and commit slim files instead of html files.
+  * html files will update automatically after merging.
```
Comment (aajisaka) [inline CONTRIBUTING.md]: will update -> will be updated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* Get two "LGTM" by organization members for merging.
```
Comment (aajisaka) [inline CONTRIBUTING.md]: At least two "LGTM" of the organization members are required for merging. かなぁ

**Comment by mikakadevo2** [comment]
Preview: http://ntt-tech-conference-01.s3-website-ap-northeast-1.amazonaws.com/40ac56f46da91ff54da9c9962f15fda9cc4db83e/

**Comment by ykubota** [comment]
Thanks comments!

**Comment by kumagi** [comment]
LGTM

**Pull request merged**